sift/src/main/java/siftscience/android/DevicePropertiesCollector.java

```diff
@@ -183,8 +184,10 @@ private AndroidDevicePropertiesJson get() {
         for (String packageName : packages) {
             try {
                 // Root app detected
-                pm.getPackageInfo(packageName, 0);
-                packagesFound.add(packageName);
+                if (pm != null) {
```

Comment:
what is the sense to check pm for null inside of the loop? can it be made outside of the loop 1 time?

Reply:
Null check shifted to outside.